player/__init__.py

```diff
@@ -0,0 +1,5 @@
+from player.base import Player
+from player.dealer import Dealer
```

Comment:
This is the only class in here that doens't have `Player` in it's name. Do we want rename this to `DealerPlayer`? It might be more consistent. However, since he dealer is a special type of player, it might make sense to keep it as is. Thoughts?